libs/openFrameworks/utils/ofUtils.h

```diff
@@ -820,6 +820,20 @@ unsigned int ofGetVersionMinor();
 /// \returns The patch version number.
 unsigned int ofGetVersionPatch();
 
+/// \brief Get the pre-release version of openFrameworks.
+///
+/// openFrameworks uses the semantic versioning system.
+///
+/// For pre-release versions of openFrameworks, including development versions,
+/// this string will describe the pre-release state. Examples might include
+/// "master", "rc1", "rc2", etc.  For all official releases, this string will be
```

Comment:
maybe replace "official" with "full" or "proper"/"main" - prereleases are also official in that we (should) decide which commit is the prerelease state, and set a git tag, etc.